### DISCUSSION TITLE
Upgrade to PHP 8

### DISCUSSION THREAD

**Discussion by rstemmer** [firefly-iii/firefly-iii#4391] [General]
Hi,

Since [Arch Linux switched to php8 as default PHP version](https://archlinux.org/news/php-80-and-php-7-legacy-packages-are-available/) I'd like to know when a FireFly version gets released that works with php8?

It was one of the applications that did not work with php8 after I updated my system.
I'm using version  5.4.6

kind regards
Ralf

**Comment by JC5** [comment]
I plan to make the next version compatible with PHP 8, and the version after that have backwards incompatible code features.
